### PR TITLE
Add 3.2 sub scenario folder structure.

### PR DESCRIPTION
**Pull request by nimanthag**

### CODE REVIEW

File: product-scenarios/3-securing-an-api-with-different-security-protocols/3.2-api-developers-want-to-secure-their-api-using-an-user-access-control-machanism/3.2.1-api-developer-wants-to-secure-api-resources-using-user-roles/src/test/java/org/wso2/am/scenario/tests/api/secure/userRoles/SecureUsingUserRolesTestCases.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    private static final String ACCESS_TOKEN = "accessToken";
+    private static final String PRODUCTION = "PRODUCTION";
+    private static final String SANDBOX = "SANDBOX";
+    private static final String APPLICATION_NAME = "Application";
+    private static final String APPLICATION_DESCRIPTION = "ApplicationDescription";
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws APIManagerIntegrationTestException {
+        apiStore = new APIStoreRestClient(storeURL);
+        apiStore.login(ADMIN_LOGIN_USERNAME, ADMIN_LOGIN_PW);
```
Comment (fazlan-nazeem) [inline product-scenarios/3-securing-an-api-with-different-security-protocols/3.2-api-developers-want-to-secure-their-api-using-an-user-access-control-machanism/3.2.1-api-developer-wants-to-secure-api-resources-using-user-roles/src/test/java/org/wso2/am/scenario/tests/api/secure/userRoles/SecureUsingUserRolesTestCases.java]: better not to use admin

Comment (fazlan-nazeem) [reply product-scenarios/3-securing-an-api-with-different-security-protocols/3.2-api-developers-want-to-secure-their-api-using-an-user-access-control-machanism/3.2.1-api-developer-wants-to-secure-api-resources-using-user-roles/src/test/java/org/wso2/am/scenario/tests/api/secure/userRoles/SecureUsingUserRolesTestCases.java]: Ack

File: product-scenarios/3-securing-an-api-with-different-security-protocols/3.2-api-developers-want-to-secure-their-api-using-an-user-access-control-machanism/3.2.1-api-developer-wants-to-secure-api-resources-using-user-roles/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Start wso2am-2.6.0 distribution by executing [APIM_HOME]/bin/wso2server.sh or [APIM_HOME]/bin/wso2server.bat 
+
+## Sample Configuration
+No additional configuration or data to be added to servers.
+
+## Deployment guidelines
+API Manager 2.6.0 deployment required. No additional artifact or data to be added to servers.
+
+## REST API (if available)
+https://docs.wso2.com/display/AM260/apidocs/store/
```
Comment (praminda) [inline product-scenarios/3-securing-an-api-with-different-security-protocols/3.2-api-developers-want-to-secure-their-api-using-an-user-access-control-machanism/3.2.1-api-developer-wants-to-secure-api-resources-using-user-roles/README.md]: Should this be publisher?

Comment (nimanthag) [reply product-scenarios/3-securing-an-api-with-different-security-protocols/3.2-api-developers-want-to-secure-their-api-using-an-user-access-control-machanism/3.2.1-api-developer-wants-to-secure-api-resources-using-user-roles/README.md]: Update to publisher

**Pull request merged**